kickoff/jsonexportl10n.py

```diff
@@ -36,7 +36,7 @@ def l10nExport(releaseName):
 def _aggregateBetaLocales(releaseTable, releaseName):
     beta_prefix = releaseName.replace("beta", "b")
 
-    releases = releaseTable.query.filter(releaseTable.name.contains(beta_prefix))
+    releases = releaseTable.query.filter(releaseTable.name.contains(beta_prefix)).filter(releaseTable._shippedAt != None)
```

Comment:
Not sure we have the right version for this:
```
    r for r in cls.query.filter(cls._submittedAt > since).filter(cls._shippedAt.isnot(None)).all()
    .tox/py27/lib/python2.7/site-packages/sqlalchemy/orm/attributes.py line 139 in __getattr__
      key)
   AttributeError: Neither 'InstrumentedAttribute' object nor 'Comparator' object has an attribute 'isnot'

```